doc/manual/configuration.rst

```diff
@@ -746,6 +746,14 @@ of the respective attributes of the recipe. For example the scripts of the
 inherited class of all steps are inserted in front of the scripts of the
 current recipe. 
 
+metaVars
```

Comment:
How about calling it `metaEnvironment`? This would be similar to `environment` and `privateEnvironment`. After all, they're just some private environment variables that can be queried.